client/lib/post-normalizer/test/post-normalizer-test.js

```diff
@@ -693,6 +693,21 @@ describe( 'post-normalizer', function() {
 				}
 			);
 		} );
+		it( 'detects trusted iframes and removes the sandbox', function( done ) {
```

Comment:
Minor: Would be nice to add a test to catch bad hosts like `evilspotify.com` but shouldn't hold up this PR.